docs/plugins/lookup_lookup.rst

```diff
@@ -490,7 +490,7 @@ Examples
         loop: "{{ query('networktocode.nautobot.lookup', 'devices',
                         api_endpoint='http://localhost/',
                         api_version='1.3',
```

Comment:
Can you please update all `api_version='1.3'` to `api_version='2.0'` in this file (x3) as well as the examples (x3) in the [lookup plugin file](https://github.com/nautobot/nautobot-ansible/blob/develop/plugins/lookup/lookup.py#L79)?

Reply:
I think this should be it.